doc/test-framework/scorecard.md

```diff
@@ -325,11 +325,157 @@ Example of a valid JSON output:
 
 **NOTE:** The `ScorecardOutput.Log` field is only intended to be used to log the scorecard's output and the scorecard will ignore that field if a plugin provides it.
 
-## Running the scorecard with a deployed CSV
-
-The scorecard can be run using only a [Cluster Service Version (CSV)][olm-csv], providing a way to test cluster-ready and non-SDK operators.
+## Running the scorecard with an OLM-managed operator
+
+The scorecard can be run using a [Cluster Service Version (CSV)][olm-csv], providing a way to test cluster-ready and non-SDK operators.
+
+Running with a CSV alone requires both the `--csv-path=<CSV manifest path>` and `--olm-deployed` flags to be set. The scorecard assumes your CSV and relevant CRD's have been deployed onto the cluster using OLM when using `--olm-deployed`.
+
+The scorecard requires a proxy container in the operator's `Deployment` pod to read operator logs. A few modifications to your CSV and creation of one extra object are required to run the proxy _before_ deploying your operator with OLM:
+
+1. Create a proxy server secret containing a local Kubeconfig:
+    1. Generate a username using the scorecard proxy's namespaced owner reference.
+          ```sh
+          # Substitute "$your_namespace" for the namespace your operator will be deployed in (if any).
+          $ echo '{"apiVersion":"","kind":"","name":"scorecard","uid":"","Namespace":"'${your_namespace}'"}' | base64 -w 0
+          eyJhcGlWZXJzaW9uIjoiIiwia2luZCI6IiIsIm5hbWUiOiJzY29yZWNhcmQiLCJ1aWQiOiIiLCJOYW1lc3BhY2UiOiJvbG0ifQo=
+          ```
+    1. Write a `Config` manifest `scorecard-config.yaml` using the following template, substituting `${your_username}` for the base64 username generated above:
+          ```yaml
+          apiVersion: v1
+          kind: Config
+          clusters:
+          - cluster:
+              insecure-skip-tls-verify: true
+              server: http://${your_username}@localhost:8889
```

Comment:
Is not here to add the master URL? I mean the URL of the cluster? If yes, IMHO it can lead the user to add the wrong value since will be not always @localhost:8889 or http. WDYT about something as

```suggestion
              server: <CLUSTER_URL>
              
              ....
              
              **NOTE:** CLUSTER_URL is XXX. For example X. 
```

Reply:
The server is not your cluster URL. This is a scorecard proxy server for the cluster.

Reply:
So, could we add an explanation over what is the server and how the steps to get this info? WDYT?

Reply:
We could, but its an implementation detail that is irrelevant to users. Its an internal proxy that the user will never have to worry about.